test/e2e/provisioning/all_e2e_test.go

```diff
@@ -500,9 +504,11 @@ func TestUbuntuProvisioningWithUpgradeE2E(t *testing.T) {
 	osUsername := os.Getenv("OS_USERNAME")
 	osTenant := os.Getenv("OS_TENANT_NAME")
 	osNetwork := os.Getenv("OS_NETWORK_NAME")
+	osInstanceReadyCheckPeriod := os.Getenv("INSTANCE_READY_CHECK_PERIOD")
+	osInstanceReadyCheckTimeout := os.Getenv("INSTANCE_READY_TIMEOUT")
 
-	if osAuthURL == "" || osUsername == "" || osPassword == "" || osDomain == "" || osRegion == "" || osTenant == "" {
-		t.Fatal("unable to run test, all of OS_AUTH_URL, OS_USERNAME, OS_PASSOWRD, OS_REGION, OS_TENANT and OS_DOMAIN must be set!")
+	if osAuthURL == "" || osUsername == "" || osPassword == "" || osDomain == "" || osRegion == "" || osTenant == "" || osInstanceReadyCheckPeriod == "" || osInstanceReadyCheckTimeout == "" {
+		t.Fatal("unable to run test suite, all of OS_AUTH_URL, OS_USERNAME, OS_PASSOWRD, OS_REGION, OS_TENANT OS_DOMAIN, INSTANCE_READY_CHECK_PERIOD, and INSTANCE_READY_TIMEOUT must be set!")
```

Comment:
I don't think this should be true, those timeouts can and should be defaulted to some value (like they do in normal flow) so they can avoid being required.